contributed_definitions/nyaml/NXbias_spectroscopy.yaml

```diff
@@ -1,114 +1,92 @@
 category: base
 doc: |
-  Base classes definition for bias spectroscopy.
+  Base class definition for bias spectroscopy.
   
-  Bias sweeps while measuring arbitrary channels with I(V) curves.
+  The Scanning Tunneling Spectroscopy (STS), an extension of Scanning Tunneling Microscopy (STM),
```

Comment:
is not it the opposite?

Reply:
Updated!

Reply:
I disagree on this one. Isn't STS just one mode in which an STM machine can be operated? So I would rather say something like "Scanning tunneling spectroscopy (STS), an operation mode in Scanning Tunneling Microscopy (STM), is used to measure I(V) curves at each point of the surface..."

Reply:
STS, STM and AFM all of are parts of SPM technique. The Doc is the updated one. Now it is modified.